test/ex_aws/rekognition_test.exs

```diff
@@ -2,7 +2,21 @@ defmodule ExAws.RekognitionTest do
   use ExUnit.Case, async: true
   doctest ExAws.Rekognition
 
-  alias ExAws.Rekognition.S3Object
+  # alias ExAws.Rekognition.S3Object
```

Comment:
It's good practice to not commit commented code.

Reply:
Yes, my intent is to leave an example with the S3 object but it requires a specific bucket to exist which will probably make the tests fail for the rest of the people... But I would like to leave an example for them to see :/